### PR TITLE
CB-20884 add a skip window for core statuschecker to avoid CM related issues

### PR DESCRIPTION
**Pull request by sodre90**
When we do a status check, we will query the latest flow log end time, and if it is not older than 2 minutes, we will skip the status check. It will help in cases like the following: we did a CM management restart service, and the CM operation finished, but the services not properly started, and they need a few more seconds to start properly.

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java [hunk 1]
```diff
@@ -166,6 +178,27 @@ protected void executeTracedJob(JobExecutionContext context) throws JobExecution
...
+            LOGGER.debug("StackStatusCheckerJob cannot run, because flow is running for stack: {}", getStackId());
+            return true;
+        }
+        Optional<FlowLog> lastFlowLog = flowLogService.getLastFlowLogWithEndTime(getStackId());
+        if (lastFlowLog.isPresent()) {
+            FlowLog flowLog = lastFlowLog.get();
+            Instant skipTimeInstant = clock.nowMinus(Duration.ofMinutes(skipWindow));
+            Instant lastFlowLogEndTimeInstant = Instant.ofEpochMilli(flowLog.getEndTime());
+            if (lastFlowLogEndTimeInstant.isAfter(skipTimeInstant)) {
+                LOGGER.debug("Last flow log endtime in UTC: {}", lastFlowLogEndTimeInstant.atZone(ZoneOffset.UTC));
```
Comment (lturcsanyi) [inline core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java:192]: Do we really need these log messages that only contain a few UTC timestamps? The last log message + skipWindow seems enough.

File: integration-test/scripts/create-cloudbreak-context.sh [hunk 2]
```diff
@@ -11,6 +11,7 @@ echo -e "\n" >> integcb/Profile
 echo "export VAULT_AUTO_UNSEAL=true" >> integcb/Profile
 echo "export VAULT_DB_SCHEMA=inet_vault_$(date +%s)" >> integcb/Profile
 echo "export CB_UI_MAX_WAIT=600" >> integcb/Profile
+echo "export CB_JAVA_OPTS=-Dcb.statuschecker.skip.window.minutes=0"
```
Comment (lturcsanyi) [inline integration-test/scripts/create-cloudbreak-context.sh:14]: Please export CB_STATUSCHECKER_SKIP_WINDOW_MINUTES instead.

File: core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java (hunk 1 above)
Comment (sodre90) [reply core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java:192]: I added these for a debugging reason to see if the calculation is wrong.

Comment (lturcsanyi) [reply core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java:192]: Ok, but hopefully you have tested the calculation logic and everything worked well and you can remove these :)

File: integration-test/scripts/config-ums.sh [hunk 3]
```diff
@@ -20,16 +20,14 @@ validate-ums-users() {
 
 prepare-cb-profile() {
     echo "Replacing UMS_HOST in profile file"
-    sed '/export UMS_HOST/d' ./integcb/Profile > ./integcb/Profile.tmp1
-    sed '/export UMS_PORT/d' ./integcb/Profile > ./integcb/Profile.tmp1
-    sed '/export CB_JAVA_OPTS/d' ./integcb/Profile.tmp1 > ./integcb/Profile.tmp
-    mv ./integcb/Profile.tmp ./integcb/Profile
+    sed -i '/export UMS_HOST/d' ./integcb/Profile
```
Comment (lnardai) [inline integration-test/scripts/config-ums.sh:23]: Still not sure why I wrote this the way is with the multiple Profile.tmp files. Gues we'll find out.

**Review by lnardai — APPROVED** [review]

**Pull request merged**